yajsapi/executor/ctx.ts

```diff
@@ -293,13 +296,13 @@ export class WorkContext {
   private _started: boolean;
   private _emitter: Callable<[StorageEvent], void> | null;
   private _network_node?: NetworkNode;
-  public provider_info: { provider_id: string, provider_name: string };
+  public provider_info: { provider_id: string; provider_name: string };
```

Comment:
hmm.... can you separate dictionary values with semicolons indeed?

Reply:
ts suggests using `type` instead of an object (dictionary).
the `type` construct uses semicolons. https://www.typescriptlang.org/docs/handbook/2/objects.html